counterexamples/seminorm_lattice_not_distrib.lean

```diff
@@ -0,0 +1,95 @@
+/-
+Copyright (c) 2022 Pierre-Alexandre Bazin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pierre-Alexandre Bazin
+-/
+import analysis.seminorm
+/-!
+# The lattice of seminorms is not distributive
+
+We provide an example of three seminorms over the ℝ-vector space ℝ×ℝ whih dont verify the lattice
+distributivity property (p ⊔ q1) ⊓ (p ⊔ q2) ≤ p ⊔ (q1 ⊓ q2).
+
+This proves the lattice (seminorm ℝ ℝ×ℝ) is not distributive.
+
+## References
+
+* https://en.wikipedia.org/wiki/Seminorm#Examples
+-/
+
+private lemma bdd_below_range_add {𝕜 E : Type*} [normed_field 𝕜] [add_comm_group E] [module 𝕜 E]
+  (x : E) (p q : seminorm 𝕜 E) :
+  bdd_below (set.range (λ (u : E), p u + q (x - u))) :=
+by { use 0, rintro _ ⟨x, rfl⟩, exact add_nonneg (p.nonneg _) (q.nonneg _) }
+
+noncomputable def p : seminorm ℝ (ℝ×ℝ) :=
+{ to_fun := λ x, |x.fst| ⊔ |x.snd|,
```

Comment:
Can you construct this more readily as `norm.comp fst ⊔ norm.comp snd`, which avoids having to prove anything?